scripts/menu.gsc

```diff
@@ -653,6 +654,7 @@ MenuOptionsPlayer(menu, player)
             }
             else if(BO4GetMap() == "IX"){
                 self addOpt("Serket Kiss", &GiveClientWeapon, "ww_crossbow_t8_upgraded", self);
+                self addOpt("Iron Bull", &GiveClientWeapon, "zhield_zword_dw_upgraded", self);
```

Comment:
Since Iron Bull was added under Upgraded specialists. The Hells Reedemer (487) and Attuned Spectral Shield (492) might want to be moved to this section as well.

Reply:
Done, Good catch cause I shouldve done that already. Nice work on the update btw. Didn't even think of the ee_song_2.